stock_available_to_promise_release/models/stock_picking.py

```diff
@@ -35,3 +133,30 @@ def _release_link_backorder(self, origin_picking):
             )
             % (self.id, self.name)
         )
+
+    def _after_release_update_chain(self):
+        """Called after the moves are released
+
+        ``self`` contains all the stock.picking of the chain up
+        to the released one.
+        """
+        self._after_release_set_printed()
+        self._after_release_set_expected_date()
+
+    def _after_release_set_printed(self):
+        self.filtered(lambda p: not p.printed).printed = True
+
+    def _after_release_set_expected_date(self):
+        prep_time = self.env.company.stock_release_max_prep_time
+        new_expected_date = fields.Datetime.now() + timedelta(minutes=prep_time)
```

Comment:
We could use the Odoo API to avoid importing `timedelta`?
```
new_expected_date = fields.Datetime.add(fields.Datetime.now(), minutes=prep_time)
```

Reply:
I wasn't aware of `Datetime.add`, I'll change